src/main/scala/is/hail/expr/Type.scala

```diff
@@ -1239,6 +1239,24 @@ object TGenotype {
       "pl" -> TArray(TInt32()))
     if (required) (!t).asInstanceOf[TStruct] else t
   }
+  
+  def representationWithVCFAttributes(required: Boolean = false): TStruct = {
+    val t = TStruct(IndexedSeq(
+      Field("GT", TCall(), 0,
+        Map("Number" -> "1", "Type" -> "String", "Description" -> "Genotype")),
+      Field("AD", TArray(TInt32()), 1,
+        Map("Number" -> "R", "Type" -> "Integer",
+          "Description" -> "Allelic depths for the ref and alt alleles in the order listed")),
+      Field("DP", TInt32(), 2,
+        Map("Number" -> "1", "Type" -> "Integer", "Description" -> "Read Depth")),
+      Field("GQ", TInt32(), 3,
+        Map("Number" -> "1", "Type" -> "Integer", "Description" -> "Genotype Quality")),
+      Field("PL", TArray(TInt32()), 4,
+        Map("Number" -> "G", "Type" -> "Integer",
+          "Description" -> "Normalized, Phred-scaled likelihoods for genotypes as defined in the VCF specification"))))
+    
+    if (required) (!t).asInstanceOf[TStruct] else t
```

Comment:
this can now be t.setRequired(required). (just added the function because this seemed to be cropping up a bunch)